pkg/gce-pd-csi-driver/controller.go

```diff
@@ -328,11 +343,40 @@ func (gceCS *GCEControllerServer) createVolumeInternal(ctx context.Context, req
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument, "failed to extract parameters: %v", err.Error())
 	}
-	// Validate multiwriter
```

Comment:
Please add this comment back

Reply:
Done

Reply:
The comment should go above getMultiWriterFromCapabilities()

Reply:
Done

Reply:
The comment is still not above getMultiWriterFromCapabilities(). It should go above getMultiWriterFromCapabilities, not validateVolumeCapabilities

Reply:
Yes, I have the fix committed, but I am waiting on the current PR test to finish. I will push the change once the integration tests finishes. Sorry for the confusion